### ISSUE TITLE
When using a predefined value, hue is not set correctly.

### ISSUE THREAD

**Issue by JDemler** [simonh1000/elm-colorpicker#4]
Thanks for this ColorPicker. I really like it.

When not starting with white or black though, the hue in the colorpicker is incorrect:

[Hue should be at red, is at cyan](https://ellie-app.com/45y9CVB2VMha1)

Here the relevant code:

```elm
type alias Model =
    { colorPicker : ColorPicker.State }


initialModel : Model
initialModel =
    { colorPicker = ColorPicker.empty }


type Msg = ColorPickerMsg ColorPicker.Msg


update : Msg -> Model -> Model
update msg model =
    model


view : Model -> Html Msg
view model =
    div []
        [ Html.map ColorPickerMsg <| ColorPicker.view (Color.rgb 1.0 0.0 0.0) model.colorPicker
        ]
```

**Comment by simonh1000** [comment]
Thanks for the feedback - I'll try to look at this at the weekend

**Comment by simonh1000** [comment]
this should be fixed in 1.1.5

**Comment by JDemler** [comment]
Thanks for the quick fix!
